### PR TITLE
Bugfix/issue 42/UI fix and reorg

### PR DESCRIPTION
**Pull request by estchang**
# Issue #42 

## Description

- Fix dropdowns in search displaying icons twice
- Redesign UI to be cleaner and add labels for clarity

## Change Type (delete non-relevant options)

- 🪲 Bug fix (non-breaking change which fixes an issue)
- 🔄 Refactor (for code refactoring and tech debt)

## Dependencies

- #137 for test fix

### CODE REVIEW

File: src/donor_dashboard/templates/donor_dashboard/list.html [hunk 1]
```diff
@@ -169,4 +169,21 @@ <h2 class="subtitle mt-5">
         </div>
         </div>
     </div>
+    <script>
```
Comment (arsalananwar11) [inline src/donor_dashboard/templates/donor_dashboard/list.html:172]: Can you please shift this to a new Javascript file under static?

**Review by arsalananwar11 — CHANGES_REQUESTED** [review]
Everything else looks great Estella, thank you!

File: src/donor_dashboard/templates/donor_dashboard/list.html (hunk 1 above)
Comment (estchang) [reply src/donor_dashboard/templates/donor_dashboard/list.html:172]: I think this has been done in develop! Mine is just outdated; I'll remove the snippet.

Comment (arsalananwar11) [reply src/donor_dashboard/templates/donor_dashboard/list.html]: Sure, thanks

Comment (estchang) [reply src/donor_dashboard/templates/donor_dashboard/list.html]: @arsalananwar11 If all looks good could you resolve the requested changes? I'll merge this one and continue refactoring in my other PR!

**Review by arsalananwar11 — APPROVED** [review]
Awesome work Estella, really like the work you do!

**Pull request merged**